content/design-systems-for-developers/react/en/distribute.md

```diff
@@ -58,18 +58,18 @@ export * from './Icon';
 export * from './Link';
 ```
 
-Let’s add a development dependency on `@babel/cli` to compile our JavaScript for release:
+Let’s add a development dependency on `@babel/cli` and `cross-env` to compile our JavaScript for release:
 
 ```bash
-yarn add --dev @babel/cli
+yarn add --dev @babel/cli cross-env`
```

Comment:
@tmeasday if you don't mind, drop the extra tick after cross-env

Reply:
```suggestion
yarn add --dev @babel/cli cross-env
```